src/Analytics/Plausible.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Utopia PHP Framework
+ *
+ * @package Analytics
+ * @subpackage Tests
+ *
+ * @link https://github.com/utopia-php/framework
+ * @version 1.0 RC1
+ * @license The MIT License (MIT) <http://www.opensource.org/licenses/mit-license.php>
+ */
+
+namespace Utopia\Analytics;
+
+class  PlausibleAdapter extends Analytics
+{
+  private const URL='https://plausible.io/api/event';
+
+
+  public function __construct(string $clientIP){
+    $clientIP=$_SERVER['REMOTE_ADDR'];
```

Comment:
`$_SERVER['REMOTE_ADDR']` will only work in Server mode - but not in CLI.

For CLI mode we can fallback to `127.0.0.1`.
```suggestion
  public function __construct(string $clientIP = '127.0.0.1'){
```